src/reformat_libyuv.c

```diff
@@ -973,12 +973,12 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb,
                                                        /*attentuate=*/0);
         *alphaReformattedWithLibYUV = AVIF_TRUE;
     } else {
-        avifImage image_downshifted_to_8bit;
+        avifImage imageDownshiftedTo8bit;
         const avifImage * image8;
         if (avifImageUsesU16(image)) {
             const avifBool downshiftAlpha = (lcf.yuvaToRgbMatrixFilter != NULL || lcf.yuvaToRgbMatrix != NULL);
-            AVIF_CHECKRES(avifImageDownshiftTo8bpc(image, &image_downshifted_to_8bit, downshiftAlpha));
-            image8 = &image_downshifted_to_8bit;
+            AVIF_CHECKRES(avifImageDownshiftTo8bpc(image, &imageDownshiftedTo8bit, downshiftAlpha));
```

Comment:
Please change the second parameter of `avifImageDownshiftTo8bpc()` to the same name, so that the meaning of `image8` in this file doesn't diverge into two.

Optional: "imageDownshiftedTo8bit" could be shortened to "imageDownshifted", because "To8bit" is clear from the context.